### PR TITLE
feat(column sorting): set default sort direction to descending

### PR DESCRIPTION
**Pull request by john-traas**
This sets the default sorting direction, when clicking on the tabulator in the table view, to descending. 

Fixes https://github.com/Lundalogik/crm-feature/issues/3655

### CODE REVIEW

File: src/components/table/columns.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export class ColumnDefinitionFactory {
             field: column.field,
             hozAlign: column.horizontalAlign,
             headerSort: column.headerSort,
+            headerSortStartingDir: 'desc',
```
Comment (vicgeralds) [inline src/components/table/columns.ts:27]: Do we want to change the default in lime-elements without providing a way to change it? Also seems like a breaking change 🤔 

I think we rather want to add `headerSortStartingDir` to our `Column` type and set that instead (possibly depending on the field type) when passing the `columns` prop the the table component.

Or maybe call it `headerSortDescending: boolean` instead 🙂

**Review by vicgeralds — COMMENTED** [review]
Good job finding how to change this!

I still think limel-table needs to flexible and allow different sort starting directions, not just change it to descending always. So we need a way to pass the desired sort starting direction in `columns` I think.

**Comment by john-traas** [comment]
> Good job finding how to change this!
> 
I can't take the credit, I was close but in the end @LawrenceBorst found where the trail eventually led 😅

File: src/components/table/columns.ts (hunk 1 above)
Comment (john-traas) [reply src/components/table/columns.ts:27]: I agree, in order to implement it for specific field types we'd have to do that. I'll give that approach a try. 😃 

This first approach won't set the default sorting on columns to descending when the table is loaded, it simply ensures that descending is the first direction the tabulator sets on a column when the user clicks on it to sort it.

Comment (vicgeralds) [reply src/components/table/columns.ts:27]: > I'm not sure if this approach is a breaking changes as it won't set the default sorting on columns to descending when the table is loaded, it simply ensures that descending is the first direction the tabulator sets on a column when the user clicks on it to sort it.

You're right. It's probably not a big deal to change that default behaviour. 🙂

**Review by vicgeralds — APPROVED** [review]
I encourage you try the approach that enables passing a sorting starting direction per column, but I'm still ready to approve this if it turns out too difficult or you don't have the time. 🙂

**Comment by john-traas** [comment]
On speaking with Tomas about this the other day, I think we can go ahead and merge this one. I'll create a separate issue for handling it for specific fields, I'll explore that at the start of cooldown and if it's a simple fix, we can implement it, but if not then Tomas has said that we can wait for an upcoming pitch regarding custom layouts that Product wants us to build.

**Pull request merged**